config/initializers/sentry.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sentry.init do |config|
+  return unless ENV.has_key?('SENTRY_DSN')
+  config.dsn = ENV.fetch('SENTRY_DSN')
+  config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+  config.enable_tracing = true
```

Comment:
Let's not enable tracing via Sentry at this time. We should get the same/similar data via Scout. If we want to move that to Sentry instead we can discuss, but don't need to have both on for now.